src/components/shared/HelpText/index.tsx

```diff
@@ -4,13 +4,18 @@ import classnames from 'classnames';
 import './index.scss';
 
 type HelpTextProps = {
+  id?: string;
```

Comment:
This prop is needed to associate field with help text. We probably need to go back and fix other fields in the future.